packages/react/src/components/Geo/types/maplibre-gl-geocoder.d.ts

```diff
@@ -169,5 +169,25 @@ declare module 'maplibre-gl-geocoder' {
      * If a `bbox` is found the map will fit to the `bbox`.
      */
     zoom?: number;
+    /**
+     * Emitted when the input is cleared
+     */
+    onClear?: Function;
```

Comment:
Thank you @calebpollman, I'll certainly have a stab at these.  React isnt my background, but the guidance is great to learn.  Hope this is ok :) i'll make some changes and revert back once done.

Reply:
@calebpollman I've submitted some updates aligned with your comments, it would be great if you could take a look and let me know what you think? thanks